src/System.Linq.Parallel/tests/QueryOperators/AggregateTests.cs

```diff
@@ -304,6 +304,17 @@ public static void Aggregate_OperationCanceledException(Labeled<ParallelQuery<in
         }
 
         [Theory]
+        [MemberData(nameof(UnorderedSources.Ranges), new[] { 128 }, MemberType = typeof(UnorderedSources))]
```

Comment:
> Do you want me to reduce to fewer/single sources everywhere in this PR (where possible), or do should that be a new one?

In this PR how about we just not add new unnecessary ones.  A future PR can reduce it for existing ones as is appropriate.  Sound good?